person_manager.rb

```diff
@@ -0,0 +1,24 @@
+class PersonManager
+  attr_reader :people
+
+  def initialize
+    @people = []
+  end
+
+  def create_student(name)
+    person = Student.new(name)
+    @people << person
+    puts "#{person.name} created as a Student."
+  end
```

Comment:
- [ ] Hi there! 👋 I've noticed a small issue that needs to be addressed. When I choose the option to create a student, I'm only prompted for the student's name. However, according to the initial definition of the Student class, it should actually accept three attributes: age, name, and parent_permission. 🧐
- [ ] To ensure your project meets the specified requirements, it's important to modify your code so that when you create a new student, you're prompted to provide values for all three attributes: age, name, and parent_permission. 📝💡 This will help your project align with its intended design and functionality. Feel free to make these adjustments to ensure your app works as expected! 😊🛠️